### ISSUE TITLE
2.7.3 - Overly Skittish Workers

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#9413]
_1. Mod Version (X.Y.Z). Current Version: 2.7.3_

2.7.3

_2. DirectX Version_

_3. List of Other Mods_

_4. Describe the Issue_

Workers are supposed to stop working when barbarians are near enough to snipe them, as a QoL for the player. However, teh workers have gotten too skittish. Here is one example:

![image](https://user-images.githubusercontent.com/20593976/202601615-95705a9b-35c3-4b7d-a437-1c6d4104681d.png)

Though that spear is near, it cannot touch my workers, they are perfectly safe. But they will stop work turn after turn just because of the proximity.

**Comment by azum4roll** [comment]
I don't think it accounts for ZOC. Which is fair, since a unit providing ZOC can be killed.
